pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -336,12 +344,16 @@ let
           ln -sfT "$target" "$out/$l"
         done
 
+      '' + lib.optionalString stdenv.hostPlatform.isDarwin ''
+        # MacOS bundles are not playing well with symlinks that point outside the bundle.
```

Comment:
I'm not sure this is entirely true; I was able to make it work (at least it didn't crash) if I "only" copied Resources directory. Which is still huge (80mb) but not as huge as the whole bundle (which is ~270mb). While everything looked like it worked with just Resources copied, I don't know if it's safe. Would be nice if someone with some knowledge on how bundles for MacOS work could confirm that symlinks are even supposed to work.